ui/prometheus-plugin/src/plugins/prometheus-variables.tsx

```diff
@@ -51,7 +51,7 @@ export const PrometheusLabelNamesVariable: VariablePlugin<PrometheusLabelNamesVa
     };
   },
   dependsOn: () => [],
-  OptionsEditorComponent: () => null,
+  OptionsEditorComponent: JSONSpecEditor,
```

Comment:
Wondering if this should be the default when a plugin doesn't provide React components

Reply:
Yeah this makes sense to me!